libraries/TGRSIFormat/TChannel.cxx

```diff
@@ -401,8 +401,8 @@ double TChannel::CalibrateENG(double charge) {
    //integration parameter.
    if(ENGCoefficients.size()==0)
       return charge;
-   double cal_chg = 0.0;
-   for(int i=0;i<ENGCoefficients.size();i++){
```

Comment:
Yeah one would assume -o2 would be good enough to optimize it away. I was just being a smart ass.

> On Jul 13, 2015, at 4:44 PM, Vinzenz Bildstein <notifications@github.com> wrote:
> 
> In libraries/TGRSIFormat/TChannel.cxx:
> 
> > @@ -401,8 +401,8 @@ double TChannel::CalibrateENG(double charge) {
> >     //integration parameter.
> >     if(ENGCoefficients.size()==0)
> >        return charge;
> > -   double cal_chg = 0.0;
> > -   for(int i=0;i<ENGCoefficients.size();i++){
> You are right, the change wasn't really something necessary. Though as far as I know gcc manages i++ vs ++i well, as long as i is a simple number (not a class). But that is exactly the reason why most loops I write have an ++i at the end ;-)
> 
> —
> Reply to this email directly or view it on GitHub.
>